lua/wire/stools/hudindicator.lua

```diff
@@ -213,6 +205,19 @@ if (CLIENT) then
 		if (isregistered) then
 			draw.WordBox(8, ScrW() / 2 + 10, ScrH() / 2 + 10, "Registered", "Default", Color(50, 50, 75, 192), Color(255, 255, 255, 255))
 		end
+
+		-- Copied from WireToolSetup.SetupLinking()
```

Comment:
I'd do it the other way around. Use SetupLinking, and then override the DrawHUD function to add back in the stuff above.

Reply:
Done.